index-fastopt.html

```diff
@@ -9,7 +9,7 @@
 <h1>Example Scala.js application - fast-optimized version</h1>
 
 <p>After having compiled and fast-optimized properly the code for the application
-(using `sbt fastOptJS`), you should see "It works" herebelow.
+(using `sbt fastOptJS`), you should see "It works" here below.
```

Comment:
I'm not a native English speaker, but I'm pretty sure ["here below"](http://www.yourdictionary.com/here-below) doesn't have the appropriate meaning.

From what I can find on the net, "herebelow" is correct but rare English. I guess nowadays only French native speakers like me use it, as translation for "ci-dessous". I guess the appropriate modern word would simply be "below".